packages/devtools_app/lib/src/screens/memory/panes/diff/controller/diff_pane_controller.dart

```diff
@@ -9,10 +9,15 @@ import 'package:flutter/material.dart';
 import '../../../../../primitives/utils.dart';
 import '../../../../../shared/common_widgets.dart';
 import '../../../primitives/memory_utils.dart';
+import '../../../shared/heap/model.dart';
 import 'model.dart';
 
 class DiffPaneController {
-  final scrollController = ScrollController();
+  DiffPaneController(this.snapshotTaker);
+
+  final listScrollController = ScrollController();
```

Comment:
see my comments on https://github.com/flutter/devtools/pull/4475 about where this should be created and used.